src/amo/sagas/reviews.js

```diff
@@ -325,14 +326,17 @@ function* fetchReview({
 
     const params: GetReviewParams = {
       apiState: state.api,
-      errorHandler,
       reviewId,
     };
 
+    console.log('---- in saga, params: ', params);
+
     const response: ExternalReviewType = yield call(getReview, params);
+    console.log('---- in saga, response: ', response);
 
     yield put(setReview(response));
   } catch (error) {
+    console.log('---- in saga, error: ', error);
```

Comment:
I don't think we want to keep those `console.log` calls :)

Reply:
Why not? ;-)